keyboards/mechlovin/infinity875/rev1/info.json

```
@@ -0,0 +1,80 @@
+{
+    "usb": {
+        "pid": "0x0875",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "custom_lite": true
```

```suggestion
        "custom": true,
        "custom_lite": true
```
Should be the last one.